Error `The JSON response could not be parsed`

## Description
I get an error in the panel when typing more than a few characters into a writer field. 
The browser tab gets unresponsive and the input get's laggy. 

It seems that it has something to do with saving versions in the background. 

**Expected behavior**  
Fast input, no lag, no error message. 

**Screenshots**  
![Image](https://github.com/user-attachments/assets/2ac9bda8-a20f-42b6-9124-c1502ee225a7)


## To reproduce

1. Create a blueprint with nested blocks containing a writer field. 
2. Create a page with this template and create some blocks with nested blocks. 
3. Type in a lot of characters very fast. Error happens after entering approx. 5-10 characters. 
5. See error in the screenshot above. 


## Your setup
**Kirby Version**  
5.0.1

**Plugins**  
none

**Console output**  
```
index.min.js:2 ku: Invalid JSON response
    at vu (https://devkit.test/media/panel/19bbdec…/js/index.min.js:2:386375)
    at async https://devkit.test/media/panel/19bbdec…/js/index.min.js:2:388598
    at async s.request (https://devkit.test/media/panel/19bbdec…/js/index.min.js:2:388130)
    at async Object.save (https://devkit.test/media/panel/19bbdec…/js/index.min.js:2:392988)
Caused by: AbortError: The user aborted a request.
error	@	index.min.js:2
```


**Your system (please complete the following information)**
 - Device: Mac Book Air M1, 2020 
 - OS: macOs 15.5
 - Browser: Google Chrome  
 - Version: 138.0.7204.92 (Official Build) (arm64)

--- a/panel/src/panel/request.js
+++ b/panel/src/panel/request.js
@@ -139,6 +139,10 @@ export const responder = async (request, response) => {
 		response.text = await response.text();
 		response.json = JSON.parse(response.text);
 	} catch (error) {
+		if (error.name === "AbortError") {
+			throw error;
+		}
+
 		throw new JsonRequestError("Invalid JSON response", {
 			cause: error,
 			request,

